tasks/src/text-to-video/data.ts

```diff
@@ -0,0 +1,57 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+        {
+			description: "(Microsoft Research Video to Text) is a large-scale dataset for the open domain video captioning, which consists of 10,000 video clips from 20 categories, and each video clip is annotated with 20 English sentences.There are about 29,000 unique words in all captions. The standard splits uses 6,513 clips for training, 497 clips for validation, and 2,990 clips for testing.",
```

Comment:
These are a bit too long, they should fit into single line, e.g. see Notes under datasets in this page: https://huggingface.co/tasks/image-segmentation

Reply:
Done. Shortened them to a single sentence with brief description